src/modules/75-cf/components/SectionToggle/__test__/SectionToggle.test.tsx

```diff
@@ -48,4 +48,25 @@ describe('SectionToggle', () => {
     expect(screen.getByText('cf.shared.segments').parentElement).toHaveClass('active')
     expect(screen.getByText('cf.shared.targets').parentElement).not.toHaveClass('active')
   })
+
+  test('it should render a tooltip when hovering over Targets button', () => {
+    renderComponent()
+    const targetsBtn = screen.getByText('cf.shared.targets')
+    fireEvent.mouseOver(targetsBtn)
+    waitFor(() => {
+      expect(document.getElementsByClassName('Tooltip--tooltipContentWrapper')).toBeInTheDocument()
```

Comment:
If you need to query by class name, try `document.querySelector`
```suggestion
      expect(document.querySelector('.Tooltip--tooltipContentWrapper')).toBeInTheDocument()
```

Reply:
I _THINK_ this will actually evaluate as true even without hovering. You need to wrap the component in the `TooltipContextProvider` and give it a dictionary of tooltip ids vs content. Check out `StringWithTooltip`'s test for an example.